src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useMemo, useState, useCallback } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { IoIosArrowDown, IoIosArrowUp } from 'react-icons/io';
+
+import List, { GroupedListProps } from '../List';
+import ToggleButton from '../ToggleButton';
+
+import styles from './styles.css';
+
+type OptionKey = string | number;
+
+interface AccordionGroupProps<D, GP, GK extends OptionKey> {
+    groupKey: GK;
+    groupIndex: number;
+    data: D[];
+    children: React.ReactNode;
+    isGroupOpen: boolean;
+    setGroupOpen: (key: GK) => void;
+    groupTitleRenderer: (props: GP) => JSX.Element;
+    groupTitleRendererParams: (key: GK, index: number, data: D[]) => GP;
+    groupTitleRendererClassName?: string;
+    groupHeaderClassName?: string;
+}
+
+function AccordionGroup<D, GP, GK extends OptionKey>({
+    data,
+    children,
+    groupKey,
+    groupIndex,
+    isGroupOpen,
```

Comment:
It's confusing if it is a function or a value from the name.
It may be better to rename this as `groupOpened` or something similar.